webpack.config.js

```diff
@@ -88,7 +88,11 @@ const webpackConfig = {
 			},
 			{
 				test: /\.(t|j)sx?$/,
-				exclude: /node_modules(\/|\\)(?!(debug))/,
+				exclude: [
+					/node_modules(\/|\\)(?!(debug))/,
+					/build/,
```

Comment:
This rule was trying to process built files as TS. These should not be included.